src/components/Search/SearchStatusBar.tsx

```diff
@@ -0,0 +1,160 @@
+    {
+        key: CONST.SEARCH.STATUS.TRIP.ALL,
+        icon: Expensicons.All,
+        text: 'common.all',
+        query: SearchUtils.buildCannedSearchQuery(CONST.SEARCH.DATA_TYPES.TRIP, CONST.SEARCH.STATUS.TRIP.ALL),
+    },
+    {
+        key: CONST.SEARCH.STATUS.TRIP.DRAFTS,
+        icon: Expensicons.Pencil,
+        text: 'common.drafts',
+        query: SearchUtils.buildCannedSearchQuery(CONST.SEARCH.DATA_TYPES.TRIP, CONST.SEARCH.STATUS.TRIP.DRAFTS),
+    },
+    {
+        key: CONST.SEARCH.STATUS.TRIP.OUTSTANDING,
+        icon: Expensicons.Hourglass,
+        text: 'common.outstanding',
+        query: SearchUtils.buildCannedSearchQuery(CONST.SEARCH.DATA_TYPES.TRIP, CONST.SEARCH.STATUS.TRIP.OUTSTANDING),
+    },
+    {
+        key: CONST.SEARCH.STATUS.TRIP.APPROVED,
+        icon: Expensicons.ThumbsUp,
+        text: 'iou.approved',
+        query: SearchUtils.buildCannedSearchQuery(CONST.SEARCH.DATA_TYPES.TRIP, CONST.SEARCH.STATUS.TRIP.APPROVED),
+    },
+    {
+        key: CONST.SEARCH.STATUS.TRIP.PAID,
+        icon: Expensicons.MoneyBag,
+        text: 'iou.settledExpensify',
+        query: SearchUtils.buildCannedSearchQuery(CONST.SEARCH.DATA_TYPES.TRIP, CONST.SEARCH.STATUS.TRIP.PAID),
+    },
+];
+
+function getOptions(type: SearchDataTypes) {
+    switch (type) {
+        case CONST.SEARCH.DATA_TYPES.INVOICE:
+            return invoiceOptions;
+        case CONST.SEARCH.DATA_TYPES.TRIP:
+            return tripOptions;
+        case CONST.SEARCH.DATA_TYPES.EXPENSE:
+        default:
+            return expenseOptions;
+    }
+}
+
+function SearchStatusBar({type, status}: SearchStatusBarProps) {
+    const {singleExecution} = useSingleExecution();
+    const styles = useThemeStyles();
+    const StyleUtils = useStyleUtils();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+    const options = getOptions(type);
+
+    return (
+        <ScrollView
+            style={[styles.flexRow, styles.mh5, styles.mb5, styles.overflowScroll, styles.flexGrow0]}
+            horizontal
+            showsHorizontalScrollIndicator={false}
+        >
+            {options.map((item) => {
+                const onPress = singleExecution(() => Navigation.setParams({q: item.query}));
+                const isActive = status === item.key;
+
+                return (
+                    <Button
+                        key={item.key}
+                        text={translate(item.text)}
+                        onPress={onPress}
+                        icon={item.icon}
+                        iconFill={isActive ? theme.success : undefined}
+                        innerStyles={!isActive && styles.bgTransparent}
+                        textStyles={!isActive ? StyleUtils.getTextColorStyle(theme.textSupporting) : undefined}
+                        hoverStyles={StyleUtils.getBackgroundColorStyle(!isActive ? theme.highlightBG : theme.border)}
+                        medium
+                    />
```

Comment:
>The hover style is still inconsistent with the tab item hover style. The icon should turn green and the text should become bold when hovering.

@Expensify/design is this something that we want to do here as well? 

>Can we reuse the TabSelectorItem component, or should we write similar logic? Do we need to write some animation code to achieve this?

My initial implementation used `TabSelectorItem` but I started running into issues with the style and that component is meant to be used with `TabSelector` which has a bunch of logic that we don't need here, so I think we should use a regular button for this.

Reply:
>The hover style is still inconsistent with the tab item hover style. The icon should turn green and the text should become bold when hovering.
>>@Expensify/design is this something that we want to do here as well?

Yes, I think we should match our other tabs' hover state as @rayane-djouah mentioned.

https://github.com/user-attachments/assets/76297fe6-a061-440a-b065-5d1cb6aeea57

Reply:
Hmm one issue that I think comes with bolding/unbolding the text without having a fixed width for the button is that all other buttons will shift with it. That works for the current tabs we have because they all take equal space. 

https://github.com/user-attachments/assets/4a36ce14-6188-4bce-85c1-977a60aa5fe5

Reply:
Discussed this in Slack and @dannymcclain and I realized that all mocks have the unhovered text as bold, so we're updating the tabs in the money request flow to be bold as well.